cmd/tendermint/commands/init.go

```diff
@@ -20,6 +22,8 @@ var InitFilesCmd = &cobra.Command{
 	RunE:  initFiles,
 }
 
+var logger = log.NewTMLogger(log.NewSyncWriter(os.Stdout))
```

Comment:
why do we need another instance of logger if we already have one in `root.go` https://github.com/tendermint/tendermint/blob/5a6822c8acbd7353729cbb5d8393de2f6da39425/cmd/tendermint/commands/root.go#L18